docs/superpowers/plans/2026-06-13-confirmation-threshold-hardening.md

```
@@ -0,0 +1,498 @@
+`npx vite-node verify-lookback.ts` → expect `ALL PASSED`.
+
+- [ ] **Step 6: Delete the script + type-check**
+
+`rm verify-lookback.ts`; then `npm run check`. The check WILL report an error in `confirmationThreshold.ts` (it still calls the old `getRecentObservedSamples`) — that is expected and fixed in Task 2. Confirm there are no OTHER new errors. (If you prefer a green check here, you may proceed to Task 2 before committing; but commit boundaries are cleaner per-task — note the expected resolver error in your report.)
+
+- [ ] **Step 7: Commit**
+
+```bash
+git add src/lib/server/db/repositories/monitoring.ts src/lib/server/db/dbimpl.ts
+git commit -m "feat(monitoring): widen confirmation lookback to include overlays + type (#756)"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)